src/azure-cli/azure/cli/command_modules/sql/tests/latest/recordings/test_metric_alert_for_sql_database_scope.yaml

```diff
@@ -0,0 +1,920 @@
+interactions:
```

Comment:
this recording should be on a different path, here:
`https://github.com/Azure/azure-cli/blob/main/src/azure-cli/azure/cli/command_modules/monitor/tests/latest/recordings/test_metric_alert_for_sql_database_scope.yaml`

Reply:
oh, my bad. Fixed it.